app-backend/api/routes/flashcardsRoute.ts

```diff
@@ -0,0 +1,15 @@
+import express from "express";
+import showFlashcardList from "./flashcardsRoutes/showFlashcardList.js";
+import showUserFlashcards from "./flashcardsRoutes/showUserFlashcards.js";
+import createFlashcardSet from "./flashcardsRoutes/createFlashcardSet.js";
+import deleteFlashcardSet from "./flashcardsRoutes/deleteFlashcardSet.js";
+import updateFlashcardSet from "./flashcardsRoutes/updateFlashcardSet.js";
```

Comment:
czemu import `.js` ?

Reply:
Ponieważ jak używasz ES modules a nie common js to musisz dodawać .js ponieważ inaczej ci nie zimportuje, a zdecydowałem się użyć ES modules aby chociażby importować paczki asynchronicznie